homeassistant/components/sia/sia_entity_base.py

```diff
@@ -81,7 +81,8 @@ async def async_will_remove_from_hass(self) -> None:
         if self._cancel_availability_cb:
             self._cancel_availability_cb()
 
-    async def async_handle_event(self, sia_event: SIAEvent) -> None:
+    @callback
+    def handle_event(self, sia_event: SIAEvent) -> None:
```

Comment:
We still want to keep the `async_*` name prefix. That's our standard for functions that should run in the event loop.

https://developers.home-assistant.io/docs/asyncio_categorizing_functions
```suggestion
    def async_handle_event(self, sia_event: SIAEvent) -> None:
```